exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -91,7 +97,22 @@ impl JavaServiceRuntime {
     }
 }
 
-/// TODO
+/// Panics if `_JAVA_OPTIONS` environmental variable is set.
+pub fn panic_if_java_options() {
+    if env::var("_JAVA_OPTIONS").is_ok() {
```

Comment:
I remember you wrote me there are several of them?

Reply:
Actually, only `_JAVA_OPTIONS` is important for us, cause only this variable can overwrite command-line args